orchestration/scripts/check-existing-file.py

```diff
@@ -0,0 +1,23 @@
+import google.auth
+from google.auth.transport.requests import AuthorizedSession
+from requests.exceptions import HTTPError
+
+credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+
+authed_session = AuthorizedSession(credentials)
+base_url = os.environ["API_URL"]
+dataset_id = os.environ["DATASET_ID"]
+target_path = os.environ["TARGET_PATH"]
+headers = {"accept": "application/json"}
+
+
+def check_file_existence(target_path: str):
+    response = authed_session.get(base_url + f"datasets/{dataset_id}/filesystem/objects",
+                                  params={"path": target_path},
+                                  headers=headers)
+    if response.ok:
+        return response.status_code
+    else:
+        raise HTTPError(f"Bad response, got code of: {response.status_code}")
```

Comment:
file exists: 200 code and a response body detailing the fileId, checksums, and other metadata
file doesn't exist: 404 code, with a message of "File not found: {path}"

I can have the script return `response.status_code == 200`

Reply:
jk we'd probably want 200 -> true, 404 -> false, and anything else to error out

Reply:
Is the 404 a piece of data within the response body, or is it the code for the entire response? If the latter, I don't think using `response.ok` like this will work. We'll need to if/else on `response.status_code`.

Reply:
the 404 is the code for the entire response. I have changed it from `response.ok` and am indeed if/else-ing on `response.status_code` 🎉